ci/certification/netgear-rax40-bhwifi.yml

```diff
@@ -0,0 +1,934 @@
+.certification:netgear-rax40-bhwifi:
+  stage: certification-netgear-rax40-bhwifi
+  extends: .certification:netgear-rax40
+
+MAP-4.2.2_BH24G:netgear-rax40-bhwifi:
```

Comment:
Actually, this is not good: it should be `MAP-4.2.2_BH24G:netgear-rax40` so without the -bhwifi, otherwise magic comment doesn't work as expected (you'd have to set MAP-4.2.2_BH24G:netgear-rax40-bhwifi instead of MAP-4.2.2_BH24G:netgear-rax40).

Reply:
Yes indeed...